lib/ruby_lsp/client_capabilities.rb

```diff
@@ -32,6 +33,9 @@ def initialize
 
       # The editor supports displaying progress requests
       @supports_progress = T.let(false, T::Boolean)
+
+      # The editor supports server initiated refresh for diagnostics
+      @diagnostic_refresh_support = T.let(false, T::Boolean)
```

Comment:
```suggestion
      @supports_diagnostic_refresh = T.let(false, T::Boolean)
```
to better match `@supports_progress`?